include/spdlog/sinks/daily_file_sink.h

```diff
@@ -46,21 +143,23 @@ class daily_file_sink final : public base_sink<Mutex>
 {
 public:
     // create daily file sink which rotates on given time
-    daily_file_sink(filename_t base_filename, int rotation_hour, int rotation_minute, bool truncate = false, uint16_t max_files = 0)
+    daily_file_sink(filename_t base_filename, int rotation_hour, int rotation_minute, bool truncate = false, uint16_t max_files = 0, bool is_fmt = false)
```

Comment:
No need another flag. this is a template class that get as a template param FileNameCalc, which can be set to fmt_filename_calc (from a new  factory function maybe to make it nicer to use)